### DISCUSSION TITLE
A tool to convert JSON Resume schema to RenderCV schema

### DISCUSSION THREAD

**Discussion by guruor** [rendercv/rendercv#150] [Show and tell]
I have recently explored RenderCV and found the templates to be minimal and the PDF friendly, thanks for the nice project @sinaatalay. 
I have been using [JSON Resume](https://github.com/jsonresume/resume-schema) for the resume generation from so long and found the schema to be pretty standard and didn't wanted to migrate to a new schema. I wanted to convert JSON Resume schema to RenderCV schema so I tried the existing package [@jsonresume/jsonresume-to-rendercv](https://www.npmjs.com/package/@jsonresume/jsonresume-to-rendercv) by @thomasdavis himself, but I found it to be not compatible with latest schema. 

Since the code was not present on GitHub to contribute, I created my own python package [jsonresume-to-rendercv](https://github.com/guruor/jsonresume-to-rendercv) for the same purpose. it would be helpful for people already using JSON resume schema.

**Comment by sinaatalay** [comment]
Thank you! I am sure this will be useful for JSON Resume users. We should consider adding this to the [FAQ](https://docs.rendercv.com/user_guide/faq/).

**Reply by guruor** [reply]
Apologies for replying to a closed discussion.
Just an update that I have created a PR to update the FAQ: https://github.com/rendercv/rendercv/pull/235

**Comment by thomasdavis** [comment]
That's awesome @guruor 

The code for that package is in the mono repo  https://github.com/jsonresume/jsonresume.org/tree/master/packages/converters/jsonresume-to-rendercv

I will update the npm page. But your python package looks way better anyway

**Reply by guruor** [reply]
Got it, Now I can see the updated link.
